### ISSUE TITLE
Pin python-version

### ISSUE THREAD

**Issue by dcramer** [trotto/go-links#81]
Dockerfile is using 3.7, readme says 3.8. Is there a suggested version and should it also be pinned via pyenv (`.python-version`)?

**Comment by itsojon** [comment]
Good question, that's confusing. All tests are run with 3.7, so I'll update the README to say 3.7.

I'd love to pin the version for the virtual env. Since there doesn't seem to be an equivalent of `.python-version` for venv, we can prescribe pyenv. Sound good to you?

**Comment by dcramer** [comment]
Yep - I can propose some other suggestions that we often use @getsentry to make devenvs easier to manage

**Comment by itsojon** [comment]
@dcramer I'd greatly appreciate that! There are still some warts in here from the closed-to-open-source switch, but it's quite stable as the core our customers are using constantly every day.
